content/general/concepts/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Environment Variables'
+Description: 'Environment variables consist of a name/value pair and are set outside of applications, but accessible to them from the environment in which they are run.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Unix'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+**Environment variables** consist of a name/value pair and are set outside of applications, but accessible to them from the environment in which they are run.
```

Comment:
Can we re-word the second half of the sentence by specifying what is accessible and what is "them." Perhaps create a second sentence.

Reply:
@yangc95 thank you for reviewing! I reworded this sentence. Hope it is OK now?